lxd/instance_backup.go

```diff
@@ -615,8 +617,11 @@ func instanceBackupDelete(d *Daemon, r *http.Request) response.Response {
 		return nil
 	}
 
-	resources := map[string][]string{}
-	resources["container"] = []string{name}
+	resources := map[string][]api.URL{}
+	resources["instances"] = []api.URL{*api.NewURL().Path(version.APIVersion, "instances", name, "backups", backupName)}
```

Comment:
This is the wrong URL, needs to point to an instance, not a backup

Reply:
In `snapshotDelete` below we include the "instances_snapshots" entry for the deleted snapshot, so we should include the "backups" entry for the deleted backup here.